docs/standard/library-guidance/nuget.md

```diff
@@ -53,6 +53,7 @@ A NuGet package supports many [metadata properties](/nuget/reference/nuspec). Th
 | `PackageIconUrl`                   | `iconUrl`                  | A URL for an image to use as the icon for the package. URL should be HTTPS and the image should be 64x64 and have a transparent background.             |
 | `PackageProjectUrl`                | `projectUrl`               | A URL for the project homepage or source repository.             |
 | `PackageLicenseUrl`                | `licenseUrl`               | A URL to the project license. Can be the URL to the `LICENSE` file in source control.             |
+| `RepositoryUrl` and `RepositoryType` | `repository url="" type=""` | A URL that can be directly used by source control software of the specified type. It should not be an HTML page. |
```

Comment:
This shouldn't be included here. It is better to setup SourceLink and allow it to add this metadata. SourceLink also includes the exact commit as well.

e.g.
![image](https://user-images.githubusercontent.com/303201/47052906-0973fb00-d207-11e8-8563-e5534d45d2b9.png)

What I think should be done:

The SourceLink recommendation should be moved down from the section above to this one, and it should have some more details included:

```
✔️ CONSIDER setting up [SourceLink](./sourcelink.md) to add source control metadata to your assemblies and NuGet package.

> SourceLink automatically adds `RepositoryUrl` and `RepositoryType` metadata to the NuGet package. SourceLink also adds an identity of the exact source code the package was built from. For example, a package created from a git repository will have the commit has added as metadata.
```

What do you think?

Reply:
Absolutely, moving the SourceLink recommendation and adding that paragraph would make much sense. 👍 

I would however suggest to keep the `RepositoryUrl` and `RepositoryType` descriptions in the table (the Nuspec column isn't very pretty unfortunately, but I'm not so sure about how to improve it). It is “Important NuGet package metadata” after all and otherwise it would never be mentioned before the SourceLink recommendation.

Reply:
This isn't a reference guide of all metadata. The linked page immediately above this table does that job.

This table is a curated list of properties you should set. Because SourceLink sets these properties for you they shouldn't be in this list.